lib/rubocop/cop/chef/sharing/include_property_descriptions.rb

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright:: Copyright 2020, Chef Software Inc.
+# Author:: Tim Smith (<tsmith@chef.io>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+module RuboCop
+  module Cop
+    module Chef
+      module ChefSharing
+        # Resources properties should include description fields to allow automated documention. Requires Chef Infra Client 13.9 or later.
```

Comment:
```suggestion
        # Resource properties should include description fields to allow automated documention. Requires Chef Infra Client 13.9 or later.
```

Should this be plural?